docs/source/production_inference.rst

```diff
@@ -28,3 +28,21 @@ Once you have the exported model, you can run it on your ONNX runtime in the fol
     input_name = ort_session.get_inputs()[0].name
     ort_inputs = {input_name: np.random.randn(1, 64).astype(np.float32)}
     ort_outs = ort_session.run(None, ort_inputs)
+
+
+Exporting to TorchScript
+------------------------
+
+TorchScript allows you to serialize your models in a way that it can be loaded in non-Python environments.
+The LightningModule has a handy method :meth:`~pytorch_lightning.core.lightning.LightningModule.to_torchscript`
+that returns a scripted module which you can save or directly use.
+
+.. code-block:: python
+
+    model = SimpleModel()
+    script = model.to_torchscript()
+
+    # save for use in production environment
+    torch.jit.save(script, "model.pt")
```

Comment:
maybe we should have a `file_path` just like in `to_onnx`?? If it's None, return the script else save it in a file.

Reply:
I think it should have a file_path but it should also always return the script (and only optionally save)

Reply:
@ananthsub thoughts on this?

Reply:
I'm fine with an optional file_path. This would provide some consistency with `to_onnx`. But I agree with @justusschock that it should always return the script module

Reply:
done, documented and tested.